Common/Sources/Helper/HyperwalletIcon.swift

```diff
@@ -44,7 +44,7 @@ public final class HyperwalletIcon {
             return .wire
 
         case HyperwalletTransferMethod.TransferMethodType.venmoAccount.rawValue:
-            return .venmo
+            return .moneygram
```

Comment:
This is not correct. Why would moneygram and venmo have the same icon? Have you tested what icon you are getting for moneygram?

Reply:
@pdamodarannair and @aparthibanpaypal kindly check with Brandon regarding this. The changes should be done in `HyperwalletIconContent` not here

Reply:
Modified as per review comments.